LICENSE.txt

```diff
@@ -0,0 +1 @@
+Satellogic SA Copyright 2017. All our code is GPLv3 licensed.
```

Comment:
This is not needed: https://github.com/satellogic/quaternions/blob/master/LICENSE

Reply:
oops spillover, will remove